### PR TITLE
Feature/heatmap click

### PR DESCRIPTION
**Pull request by Sebaxp95**

### CODE REVIEW

File: src/Spectre.AngularClient/src/app/heatmaps/heatmap/heatmap.component.ts [hunk 1]
```diff
@@ -27,6 +27,7 @@ export class HeatmapComponent implements OnInit {
   @Input() public heatmapData: any;
   @Input() public height = 600;
   @Input() public width = 600;
+  @Input() public onClickFunction: any;
```
Comment (gmrukwa) [inline src/Spectre.AngularClient/src/app/heatmaps/heatmap/heatmap.component.ts]: Is there no type for *function* specifically? I suppose I saw it below. What is the difference?

**Comment by Sebaxp95** [comment]
Do you mean disable scroll heatmap image when cursor is on it? Or you want have pan option as default? 
heatmap

**Comment by gmrukwa** [comment]
I've meant these *sliders* @danielbabiak added some time ago.

**Comment by gmrukwa** [comment]
[UX improvements in reworked GUI](https://trello.com/c/Vekb81xs/237-ux-improvements-in-reworked-gui)

**Pull request merged**